[2014-02-14 07:52:17] log.ERROR: exception 'ReflectionException' with message 'Class Bai_vietsController does not exist' in D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Container\Container.php:308
Stack trace:
#0 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Container\Container.php(308): ReflectionClass->__construct('Bai_vietsContro...')
#1 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('Bai_vietsContro...', Array)
#2 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('Bai_vietsContro...', Array)
#3 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('Bai_vietsContro...')
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#5 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\lvtn-thdhct\index.php(49): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2014-02-14 08:20:05] log.ERROR: exception 'PDOException' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'id_the_loai_bai_viet' in 'field list'' in D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Connection.php:331
Stack trace:
#0 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Connection.php(331): PDO->prepare('insert into `ba...')
#1 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Connection.php(521): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'insert into `ba...', Array)
#2 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Connection.php(332): Illuminate\Database\Connection->run('insert into `ba...', Array, Object(Closure))
#3 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Connection.php(289): Illuminate\Database\Connection->statement('insert into `ba...', Array)
#4 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Query\Processors\Processor.php(30): Illuminate\Database\Connection->insert('insert into `ba...', Array)
#5 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1444): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `ba...', Array, 'id')
#6 [internal function]: Illuminate\Database\Query\Builder->insertGetId(Array, 'id')
#7 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(786): call_user_func_array(Array, Array)
#8 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1259): Illuminate\Database\Eloquent\Builder->__call('insertGetId', Array)
#9 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1259): Illuminate\Database\Eloquent\Builder->insertGetId(Array, 'id')
#10 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1229): Illuminate\Database\Eloquent\Model->insertAndSetId(Object(Illuminate\Database\Eloquent\Builder), Array)
#11 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1140): Illuminate\Database\Eloquent\Model->performInsert(Object(Illuminate\Database\Eloquent\Builder))
#12 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(377): Illuminate\Database\Eloquent\Model->save()
#13 D:\xampp\htdocs\lvtn-thdhct\app\controllers\Bai_vietsController.php(51): Illuminate\Database\Eloquent\Model::create(Array)
#14 [internal function]: Bai_vietsController->store()
#15 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#16 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('store', Array)
#17 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'store', Array)
#18 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#19 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#20 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#21 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#22 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#23 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#24 D:\xampp\htdocs\lvtn-thdhct\index.php(49): Illuminate\Foundation\Application->run()
#25 {main}

Next exception 'Exception' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'id_the_loai_bai_viet' in 'field list' (SQL: insert into `bai_viets` (`ma_bai_viet`, `ten_bai_viet`, `noi_dung_bai_viet`, `id_nguoi_sua`, `tag`, `ghi_chu`, `id_the_loai_bai_viet`, `id_nguoi_tao_bai_viet`, `updated_at`, `created_at`) values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)) (Bindings: array (
  0 => '1',
  1 => 'Đọ cấu hình loạt ‘quái vật’ LG G Pro 2, Lumia 1520, Galaxy Note 3 và HTC One Max',
  2 => 'Đúng như những dự đoán ban đầu, LG đã chính thức công bố siêu smartphone màn hình lớn G Pro 2 với nâng cấp cấu hình vượt trội so với đàn em Optimus G Pro ra mắt cách đây không lâu. Giờ đây, cán cân phablet giữa LG, Samsung hay Nokia đã được cân bằng. Giữa LG G Pro 2, Samsung Galaxy Note 3 hay Nokia Lumia 1520, thật khó để đánh giá đâu mới là chiếc điện thoại mạnh nhất. Tất cả đã thay đổi, ranh giới chênh lệch về cấu hình gần như không còn tồn tại và ngày càng trở nên bão hòa.',
  3 => '1',
  4 => 'jkhk,asdja,asldjalksd,aljsdka',
  5 => 'null',
  6 => '1',
  7 => '1',
  8 => '2014-02-14 08:20:05',
  9 => '2014-02-14 08:20:05',
))' in D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Connection.php:556
Stack trace:
#0 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Connection.php(529): Illuminate\Database\Connection->handleQueryException(Object(PDOException), 'insert into `ba...', Array)
#1 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Connection.php(332): Illuminate\Database\Connection->run('insert into `ba...', Array, Object(Closure))
#2 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Connection.php(289): Illuminate\Database\Connection->statement('insert into `ba...', Array)
#3 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Query\Processors\Processor.php(30): Illuminate\Database\Connection->insert('insert into `ba...', Array)
#4 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1444): Illuminate\Database\Query\Processors\Processor->processInsertGetId(Object(Illuminate\Database\Query\Builder), 'insert into `ba...', Array, 'id')
#5 [internal function]: Illuminate\Database\Query\Builder->insertGetId(Array, 'id')
#6 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(786): call_user_func_array(Array, Array)
#7 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1259): Illuminate\Database\Eloquent\Builder->__call('insertGetId', Array)
#8 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1259): Illuminate\Database\Eloquent\Builder->insertGetId(Array, 'id')
#9 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1229): Illuminate\Database\Eloquent\Model->insertAndSetId(Object(Illuminate\Database\Eloquent\Builder), Array)
#10 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(1140): Illuminate\Database\Eloquent\Model->performInsert(Object(Illuminate\Database\Eloquent\Builder))
#11 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(377): Illuminate\Database\Eloquent\Model->save()
#12 D:\xampp\htdocs\lvtn-thdhct\app\controllers\Bai_vietsController.php(51): Illuminate\Database\Eloquent\Model::create(Array)
#13 [internal function]: Bai_vietsController->store()
#14 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#15 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('store', Array)
#16 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'store', Array)
#17 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#18 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#19 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#20 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#21 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#22 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#23 D:\xampp\htdocs\lvtn-thdhct\index.php(49): Illuminate\Foundation\Application->run()
#24 {main} [] []
[2014-02-14 08:23:09] log.ERROR: exception 'Illuminate\Session\TokenMismatchException' in D:\xampp\htdocs\lvtn-thdhct\app\filters.php:84
Stack trace:
#0 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#1 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(159): call_user_func_array(Object(Closure), Array)
#2 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(100): Illuminate\Routing\Route->callFilter('csrf', Object(Illuminate\Http\Request))
#3 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(43): Illuminate\Routing\Route->callBeforeFilters(Object(Illuminate\Http\Request))
#4 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 D:\xampp\htdocs\lvtn-thdhct\index.php(49): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2014-02-14 08:23:16] log.ERROR: exception 'Illuminate\Session\TokenMismatchException' in D:\xampp\htdocs\lvtn-thdhct\app\filters.php:84
Stack trace:
#0 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#1 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(159): call_user_func_array(Object(Closure), Array)
#2 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(100): Illuminate\Routing\Route->callFilter('csrf', Object(Illuminate\Http\Request))
#3 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(43): Illuminate\Routing\Route->callBeforeFilters(Object(Illuminate\Http\Request))
#4 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 D:\xampp\htdocs\lvtn-thdhct\index.php(49): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2014-02-14 08:25:15] log.ERROR: exception 'Illuminate\Session\TokenMismatchException' in D:\xampp\htdocs\lvtn-thdhct\app\filters.php:84
Stack trace:
#0 [internal function]: {closure}(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#1 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(159): call_user_func_array(Object(Closure), Array)
#2 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(100): Illuminate\Routing\Route->callFilter('csrf', Object(Illuminate\Http\Request))
#3 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(43): Illuminate\Routing\Route->callBeforeFilters(Object(Illuminate\Http\Request))
#4 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 D:\xampp\htdocs\lvtn-thdhct\index.php(49): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2014-02-14 08:36:39] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ';'' in D:\xampp\htdocs\lvtn-thdhct\app\routes.php:30
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-02-14 08:36:39] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ';'' in D:\xampp\htdocs\lvtn-thdhct\app\routes.php:30
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-02-14 08:36:46] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'Route' (T_STRING)' in D:\xampp\htdocs\lvtn-thdhct\app\routes.php:31
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-02-14 08:38:34] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'You' (T_STRING)' in D:\xampp\htdocs\lvtn-thdhct\app\filters.php:65
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-02-14 08:38:45] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'are' (T_STRING)' in D:\xampp\htdocs\lvtn-thdhct\app\filters.php:65
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-02-14 08:44:07] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function before() on a non-object' in D:\xampp\htdocs\lvtn-thdhct\app\routes.php:36
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-02-14 08:50:26] log.ERROR: exception 'Illuminate\Database\Eloquent\ModelNotFoundException' in D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php:498
Stack trace:
#0 D:\xampp\htdocs\lvtn-thdhct\app\controllers\Bai_vietsController.php(70): Illuminate\Database\Eloquent\Model::findOrFail('logout')
#1 [internal function]: Bai_vietsController->show('logout')
#2 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#3 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('show', Array)
#4 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'show', Array)
#5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('logout')
#6 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#7 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#8 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\lvtn-thdhct\index.php(49): Illuminate\Foundation\Application->run()
#12 {main} [] []
[2014-02-14 09:00:05] log.ERROR: exception 'Symfony\Component\Routing\Exception\MissingMandatoryParametersException' with message 'Some mandatory parameters are missing ("bai_viets") to generate a URL for route "bai_viets.show".' in D:\xampp\htdocs\lvtn-thdhct\vendor\symfony\routing\Symfony\Component\Routing\Generator\UrlGenerator.php:155
Stack trace:
#0 D:\xampp\htdocs\lvtn-thdhct\vendor\symfony\routing\Symfony\Component\Routing\Generator\UrlGenerator.php(140): Symfony\Component\Routing\Generator\UrlGenerator->doGenerate(Array, Array, Array, Array, Array, 'bai_viets.show', true, Array)
#1 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\UrlGenerator.php(192): Symfony\Component\Routing\Generator\UrlGenerator->generate('bai_viets.show', Array, true)
#2 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Html\HtmlBuilder.php(184): Illuminate\Routing\UrlGenerator->route('bai_viets.show', Array)
#3 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Support\helpers.php(602): Illuminate\Html\HtmlBuilder->linkRoute('bai_viets.show', '????? c???u h??...', Array, Array)
#4 D:\xampp\htdocs\lvtn-thdhct\app\storage\views\7abb61e9b420578e0310b9ef565b4ba9(6): link_to_route('bai_viets.show', '????? c???u h??...')
#5 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\xampp\htdocs...')
#6 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\xampp\htdocs...', Array)
#7 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\xampp\htdocs...', Array)
#8 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#9 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Http\Response.php(70): Illuminate\View\View->render()
#10 D:\xampp\htdocs\lvtn-thdhct\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#11 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#12 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#13 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'index', Object(Illuminate\View\View))
#14 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'index', Array)
#15 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#16 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#17 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#18 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#19 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#20 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#21 D:\xampp\htdocs\lvtn-thdhct\index.php(49): Illuminate\Foundation\Application->run()
#22 {main} [] []
[2014-02-14 09:02:52] log.ERROR: exception 'ErrorException' with message 'Undefined variable: id_bai_viet' in D:\xampp\htdocs\lvtn-thdhct\app\storage\views\7abb61e9b420578e0310b9ef565b4ba9:6
Stack trace:
#0 D:\xampp\htdocs\lvtn-thdhct\app\storage\views\7abb61e9b420578e0310b9ef565b4ba9(6): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', 'D:\xampp\htdocs...', 6, Array)
#1 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\xampp\htdocs...')
#2 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\xampp\htdocs...', Array)
#3 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\xampp\htdocs...', Array)
#4 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Http\Response.php(70): Illuminate\View\View->render()
#6 D:\xampp\htdocs\lvtn-thdhct\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'index', Object(Illuminate\View\View))
#10 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'index', Array)
#11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#12 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#13 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#14 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#15 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#17 D:\xampp\htdocs\lvtn-thdhct\index.php(49): Illuminate\Foundation\Application->run()
#18 {main} [] []
[2014-02-14 09:04:18] log.ERROR: exception 'Illuminate\Database\Eloquent\ModelNotFoundException' in D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php:498
Stack trace:
#0 D:\xampp\htdocs\lvtn-thdhct\app\controllers\Bai_vietsController.php(70): Illuminate\Database\Eloquent\Model::findOrFail('$bai_viet->id_b...')
#1 [internal function]: Bai_vietsController->show('$bai_viet->id_b...')
#2 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#3 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('show', Array)
#4 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'show', Array)
#5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('$bai_viet->id_b...')
#6 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#7 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#8 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\lvtn-thdhct\index.php(49): Illuminate\Foundation\Application->run()
#12 {main} [] []
[2014-02-14 09:07:22] log.ERROR: exception 'Illuminate\Database\Eloquent\ModelNotFoundException' in D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php:498
Stack trace:
#0 D:\xampp\htdocs\lvtn-thdhct\app\controllers\Bai_vietsController.php(70): Illuminate\Database\Eloquent\Model::findOrFail('$bai_viet->id_b...')
#1 [internal function]: Bai_vietsController->show('$bai_viet->id_b...')
#2 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#3 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('show', Array)
#4 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'show', Array)
#5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('$bai_viet->id_b...')
#6 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#7 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#8 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\lvtn-thdhct\index.php(49): Illuminate\Foundation\Application->run()
#12 {main} [] []
[2014-02-14 09:07:25] log.ERROR: exception 'Illuminate\Database\Eloquent\ModelNotFoundException' in D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php:498
Stack trace:
#0 D:\xampp\htdocs\lvtn-thdhct\app\controllers\Bai_vietsController.php(70): Illuminate\Database\Eloquent\Model::findOrFail('$bai_viet->id_b...')
#1 [internal function]: Bai_vietsController->show('$bai_viet->id_b...')
#2 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#3 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('show', Array)
#4 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'show', Array)
#5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('$bai_viet->id_b...')
#6 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#7 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#8 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\lvtn-thdhct\index.php(49): Illuminate\Foundation\Application->run()
#12 {main} [] []
[2014-02-14 09:18:57] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in D:\xampp\htdocs\lvtn-thdhct\app\views\modules\user\menu.php:12
Stack trace:
#0 D:\xampp\htdocs\lvtn-thdhct\app\views\modules\user\menu.php(12): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', 'D:\xampp\htdocs...', 12, Array)
#1 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\xampp\htdocs...')
#2 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(17): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\xampp\htdocs...', Array)
#3 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\PhpEngine->get('D:\xampp\htdocs...', Array)
#4 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\xampp\htdocs\lvtn-thdhct\app\storage\views\9a186f505e2acd4f72b2cf734a88b3fa(46): Illuminate\View\View->render()
#6 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\xampp\htdocs...')
#7 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\xampp\htdocs...', Array)
#8 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\xampp\htdocs...', Array)
#9 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#10 D:\xampp\htdocs\lvtn-thdhct\app\storage\views\7673bbe280e824619f647bf48d39ebd8(24): Illuminate\View\View->render()
#11 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\xampp\htdocs...')
#12 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\xampp\htdocs...', Array)
#13 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\xampp\htdocs...', Array)
#14 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#15 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Http\Response.php(70): Illuminate\View\View->render()
#16 D:\xampp\htdocs\lvtn-thdhct\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#17 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#18 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#19 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#20 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#21 D:\xampp\htdocs\lvtn-thdhct\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#22 D:\xampp\htdocs\lvtn-thdhct\index.php(49): Illuminate\Foundation\Application->run()
#23 {main} [] []
[2014-02-14 09:20:00] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '{'' in D:\xampp\htdocs\lvtn-thdhct\app\views\modules\user\menu.php:12
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-02-14 09:38:21] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected 'array' (T_ARRAY)' in D:\xampp\htdocs\lvtn-thdhct\app\storage\views\9a186f505e2acd4f72b2cf734a88b3fa:68
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-02-14 12:02:59] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function before() on a non-object' in D:\xampp\htdocs\lvtn-thdhct\app\routes.php:34
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
[2014-02-14 12:04:00] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function before() on a non-object' in D:\xampp\htdocs\lvtn-thdhct\app\routes.php:34
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleShutdown()
#1 {main} [] []
